api/src/backend/api/specs/v1.yaml

```diff
@@ -5856,7 +5856,6 @@ components:
               readOnly: true
               nullable: true
           required:
-          - uid
```

Comment:
I don't know why this is no longer required? 🤔

Reply:
Since TextField does not have any type of validation, Django understands this field is not required. In order to force that, you would have to update the FindingSerializer . But this is not necessary at the moment, because the Findings viewset is a readonly viewset and it doesn't make much sense to define required and non-required fields there.

Reply:
Good to know, thanks for shedding a light on this